docs/release-notes/8.0.31-23.md

```diff
@@ -0,0 +1,149 @@
+# Percona Server for MySQL 8.0.31-23 (2023-01-)
+
+<style>
+    table {
+        width=50%';
+        font-family: Chivo, Colfax, Franziska, Helvetica, Arial, sans-serif;
+    }
+    table td {
+        border: 0px;
+        padding: 8px;
+    }
+    tr:nth-child(even){
+        background-color:#f5f5f5
+    }
+    tr:hover {
+        background-color: #dddd;
+    }
+</style>
+
+| Release date | January , 2023 |
+|---|---|
+| Install instructions | [Install Percona Server for MySQL](https://docs.percona.com/percona-server/8.0/installation.html) |
+| Download this version | [Percona Server for MySQL](https://www.percona.com/downloads/Percona-Server-LATEST/)
+
+[*Percona Server for MySQL* 8.0.31-23](https://www.percona.com/software/mysql-database/percona-server) includes all the features and bug fixes available in the
+[*MySQL* 8.0.31 Community Edition](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-31.html)
+in addition to enterprise-grade features developed by Percona.
+
+*Percona Server for MySQL* is a free, fully compatible, enhanced, and open
+source drop-in replacement for any *MySQL* database. It provides superior
+performance, scalability, and instrumentation.
+
+*Percona Server for MySQL* is trusted by thousands of enterprises to provide
+better performance and concurrency for their most demanding workloads. It
+delivers more value to *MySQL* server users with optimized performance,
+greater performance scalability and availability, enhanced backups, and
+increased visibility. 
+
+For paid [support](https://www.percona.com/services/support), [managed services](https://www.percona.com/services/managed-services) or [consulting services](https://www.percona.com/services/consulting), contact [Percona Sales](https://www.percona.com/about-percona/contact)
+
+For [training](https://www.percona.com/training), contact [Percona Training - Start learning now](https://learn.percona.com/contact-me)
+
+## Release highlights
+
+Improvements and bug fixes introduced by Oracle for *MySQL* 8.0.31 and included in *Percona Server for MySQL* are the following:
+
+* Now a replica can add a Generated Invisible Primary Keys(GIPK) to any InnoDB table. To achieve this behavior, the `GENERATE` value is added as a possible value for the `CHANGE REPLICATION SOURCE TO` statement's `REQUIRE_TABLE_PRIMARY_KEY_CHECK` option.
+
+    `REQUIRE_TABLE_PRIMARY_KEY_CHECK = GENERATE` option can be used on a per-channel basis.
+
+    !!! note
+
+        A replica ignores any settings for `sql_generate_invisible_primary_key` if these settings are made on the source, and this variable is not replicated. This behavior is inherited from the previous releases.
```

Comment:
No reason for this note. Just add the information to the main text.

Reply:
Added to the main text.